drivers/hmis/app/models/hmis/role.rb

```diff
@@ -163,13 +167,87 @@ def self.permissions_with_descriptions
-        description: 'Ability to manage bed and unit capacity in the project',
+        description: 'Ability to manage bed and unit capacity in the project. If Coordinated Entry is enabled in the project, users can also mark units as available for receiving referrals.',
         administrative: false,
         access: [:editable],
         category: 'Project Access',
         sub_category: 'Management',
       },
-      can_manage_incoming_referrals: {
+      can_view_prioritized_client_lists: {
+        description: 'Ability to view client waitlists in the project',
+        administrative: false,
+        access: [:viewable],
+        category: 'Project Access',
+        sub_category: 'Referrals',
+        proc: -> { Hmis::Ce.configuration.enabled? },
+      },
+      can_start_referrals: {
+        description: 'Ability to initiate referrals from the client waitlist in the project',
+        administrative: false,
+        access: [:editable],
+        category: 'Project Access',
+        sub_category: 'Referrals',
+        proc: -> { Hmis::Ce.configuration.enabled? },
+      },
+      can_view_referrals: {
+        description: 'Ability to view all referrals to the project',
+        administrative: false,
+        access: [:viewable],
+        category: 'Project Access',
+        sub_category: 'Referrals',
+        proc: -> { Hmis::Ce.configuration.enabled? },
+      },
+      can_view_own_referrals: {
+        description: 'Grants the same permissions as "Can view referrals", but only for referrals in which this user has at least one assigned, available task',
+        administrative: false,
+        access: [:viewable],
+        category: 'Project Access',
+        sub_category: 'Referrals',
+        proc: -> { Hmis::Ce.configuration.enabled? },
+      },
+      can_perform_any_referral_tasks: {
+        description: 'Ability to start and submit any referral task, and add referral notes, on all referrals to the project',
+        administrative: true,
+        access: [:editable],
+        category: 'Project Access',
+        sub_category: 'Referrals',
+        proc: -> { Hmis::Ce.configuration.enabled? },
+      },
+      can_perform_own_referral_tasks: {
+        description: 'Grants the same permission as "Can perform any referral tasks," but only for referrals in which this user has at least one assigned, available task. This permission informs the dropdown of users available for assignment to referral swimlanes.',
+        administrative: false,
+        access: [:editable],
+        category: 'Project Access',
+        sub_category: 'Referrals',
+        proc: -> { Hmis::Ce.configuration.enabled? },
+      },
+      can_view_client_eligible_opportunities: {
+        description: 'Ability to view the Client page showing all Opportunities a client is eligible for. This is a global permission (not per-project).',
+        administrative: true,
+        global: true,
+        access: [:viewable],
+        category: 'Administration',
```

Comment:
I think this should be in the referrals category. The "administrative: true" tag marks it as administrative in the ui.